components/features/journal/editor/editor-content.tsx

```
@@ -0,0 +1,678 @@
+                    selection.addRange(range);
+                }
+            }
+        }
+    }, [blocks]);
+
+    const handleBlockAction = (action: string, blockId: string, blockType?: BlockType) => {
+        setActiveMenu(null);
+
+        switch (action) {
+            case 'ai-assistant':
+                const block = blocks.find((b) => b.id === blockId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)